javascript/packages/node/extension/extension_helpers.cpp

```diff
@@ -46,6 +46,13 @@ napi_value CreateString(napi_env env, const char* str) {
   return result;
 }
 
+napi_value CreateStringFromHbString(napi_env env, hb_string_T string) {
+  napi_value result;
+  napi_create_string_utf8(env, string.data, string.length, &result);
+  return result;
```

Comment:
Should we use `CreateString` here?
```suggestion
  return CreateString(string.data);
```

Reply:
We sadly can't use `CreateString` as it [uses `NAPI_AUTO_LENGTH`](https://github.com/marcoroth/herb/pull/749/files#diff-6f7e35780ed4dea5630d55c673c35560decf20f5c125cf895552f6da5fec0ce3R45) which expects a null terminated string [[ref]](https://nodejs.org/api/n-api.html), but our `hb_string_T`is not necessarily null terminated.

Reply:
Ah, I see. That's a good point. Thanks for the context!